packages/netlify-cms-core/src/lib/registry.js

```diff
@@ -253,7 +253,7 @@ export async function invokeEvent({ name, data }) {
         _data = { ...data, entry };
       }
     } catch (e) {
-      console.warn(`Failed running handler for event ${name} with message: ${e.message}`);
+      throw new Error(`Failed running handler for event ${name} with message: ${e.message}`);
```

Comment:
This message will propagate to the UI:
![image](https://user-images.githubusercontent.com/26760571/135327699-e0782af0-2a9e-4750-99ae-0645f31e4dac.png)

Should we remove the `try/catch` altogether so the event handler has full control of the error?

Reply:
Great Idea. Fixed.